src/nupic/algorithms/SpatialPooler.cpp

```diff
@@ -1111,18 +1111,12 @@ void SpatialPooler::inhibitColumns_(vector<Real>& overlaps,
     density = min(density, (Real) 0.5);
   }
 
```

Comment:
What happens to the member variable tieBreaker_? Is it used anymore?  If not we should remove it from the header files and hand serialization backward compatibility appropriately.  It should also be removed from initialize otherwise the compatibility tests won't pass.